### DISCUSSION TITLE
No Models are loaded when opening the main.py file

### DISCUSSION THREAD

**Discussion by D00TO** [SunOner/yolov8_aimbot#78] [General]
Model loaded. None
Aimbot is started. Enjoy!
[Right mouse button] - Aiming at the target
[F2] - EXIT

what did I do wrong?

**Comment by SunOner** [comment]
Aimbot is started. Enjoy!

**Reply by D00TO** [reply]
nope, it just don't aim at people as intended, DESTINY 2
